src/muxer.ts

```diff
@@ -101,8 +101,10 @@ export class DataChannelMuxer implements StreamMuxer {
   /**
    * Initiate a new stream with the given name. If no name is
    * provided, the id of the stream will be used.
+   * TODO does this comment make sense? The spec says the label SHOULD be an empty string
    */
   newStream (name: string = ''): Stream {
+   // TODO The spec says the label SHOULD be an empty string: https://github.com/libp2p/specs/blob/master/webrtc/README.md#rtcdatachannel-label
```

Comment:
I see your point, though the goal was to keep the implementation flexible for future usage:
> This allows future versions of this specification to make use of the RTCDataChannel label property.
I don't have a strong opinion here.  I don't believe this function is invoked directly by implementors.  As is, this seems to comply with the spec, but we could remove the parameter to be more formal about the intention?

Reply:
> As is, this seems to comply with the spec, but we could remove the parameter to be more formal about the intention?

Agree. We should remove this param.

Reply:
The param is optional and is from https://github.com/libp2p/js-libp2p-interfaces/blob/master/packages/interface-stream-muxer/src/index.ts#L30-L34 . Removing it should be okay.